cpp/test.yaml

```diff
@@ -0,0 +1,8 @@
+test:
+    tests:
+        - name: main
+          harness: test.py
```

Comment:
Maybe, I would  consider changing the name of the test from `main` to something else, the same with `test.py`